spec/requests/vendors_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "rails_helper"
+
+# NOTE:  remove header comment as we implement the tests
```

Comment:
all of these specs are skipped.  They can all be scrapped, or used if useful.

Reply:
Why not unskip them? Do they pass?

Reply:
they do NOT pass.  And getting them to pass would involve creating views, which seems silly given there are other tickets for that work.

Reply:
kk, makes sense.